app.rb

```diff
@@ -0,0 +1,189 @@
+$LOAD_PATH << '.'
+require 'classroom'
+require 'person'
+require 'teacher'
+require 'student'
+require 'book'
+require 'rental'
+require 'utils'
+
+class App
+  attr_accessor :user_input
+
+  def initialize
+    puts 'Welcome to School Library App!'
+    Utils.print_prompt
+    @user_input = gets.chomp
+    @default_classroom = Classroom.new('default-classroom')
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def user_input_valid?(user_input, arr)
+    arr.include?(user_input)
+  end
+
+  def student_info
+    age = Utils.read_age
+    name = Utils.read_name
+    has_parent_permission = Utils.read_permission == 'Y'
+    [age, name, has_parent_permission]
+  end
+
+  def teacher_info
+    age = Utils.read_age
+    name = Utils.read_name
+    specialization = Utils.read_specialization
+    [age, name, specialization]
+  end
```

Comment:
- [ ] All methods work except the method of creating a teacher. I'm having the following error when I try to create a teacher. Kindly rectify that 👍
- Error:
![Annotation 2022-05-13 082714](https://user-images.githubusercontent.com/85113567/168234207-662de993-7c0a-44d5-ba92-209ccdd2f88f.jpg)

Reply:
resolved 👍